.cirrus.yml

```diff
@@ -0,0 +1,52 @@
+FreeBSD_task:
+  freebsd_instance:
+    image_family: freebsd-12-1
+
+  install_script:
+    - pkg update
```

Comment:
According to [pkg-install](https://man.freebsd.org/pkg-install/8):
> `pkg install` first updates any out of date repository catalogues, unless `REPO_AUTOUPDATE` has been set to `NO` in `pkg.conf`.

Reply:
Oh, yes, thanks for the reminder.